cfgov/unprocessed/apps/ccdb-search/webpack-config.js

```diff
@@ -2,70 +2,72 @@
    Settings for webpack JavaScript bundling system.
    ========================================================================== */
 
-   const TerserPlugin = require( 'terser-webpack-plugin' );
+const TerserPlugin = require( 'terser-webpack-plugin' );
 
-   /* Set warnings to true to show linter-style warnings.
+/* Set warnings to true to show linter-style warnings.
       Set mangle to false and beautify to true to debug the output code. */
-   const COMMON_MINIFICATION_CONFIG = new TerserPlugin( {
-     parallel: true,
-     extractComments: false,
-     terserOptions: {
-       ie8: false,
-       ecma: 5,
-       warnings: false,
-       mangle: true,
-       output: {
-         comments: false,
-         beautify: false
-       }
-     }
-   } );
+const COMMON_MINIFICATION_CONFIG = new TerserPlugin( {
+  parallel: true,
+  extractComments: false,
+  terserOptions: {
+    ie8: false,
+    ecma: 5,
+    warnings: false,
+    mangle: true,
+    output: {
+      comments: false,
+      beautify: false
+    }
+  }
+} );
 
-   /* Commmon webpack 'module' option used in each configuration.
+/* Commmon webpack 'module' option used in each configuration.
       Runs code through Babel and uses global supported browser list. */
-   const COMMON_MODULE_CONFIG = {
-     rules: [ {
-       test: /\.js$/,
-       /* Exclude modules from transpiling.
+const COMMON_MODULE_CONFIG = {
+  rules: [ {
+    test: /\.js$/,
+
+    /* Exclude modules from transpiling.
           The below regex will match and exclude all node modules
           except those that start with `@cfpb/` or `cfpb-`.
           Regex test: https://regex101.com/r/zizz3V/9 */
-       exclude: /node_modules\/(?!(?:@cfpb\/.+|cfpb\-.+)).+/,
-       use: {
-         loader: 'babel-loader?cacheDirectory=true',
-         options: {
-           presets: [ [ '@babel/preset-env', {
-             configPath: __dirname,
-             /* Use useBuiltIns: 'usage' and set `debug: true` to see what
+    exclude: /node_modules\/(?!(?:@cfpb\/.+|cfpb-.+)).+/,
+    use: {
+      loader: 'babel-loader?cacheDirectory=true',
+      options: {
+        presets: [ [ '@babel/preset-env', {
+          configPath: __dirname,
+
+          /* Use useBuiltIns: 'usage' and set `debug: true` to see what
                 scripts require polyfilling. */
```

Comment:
Trivial: align the comment to its new indentation above.

Reply:
Fixed in 2e529be4eb50be9c927b4aeb046a8eee6bddb9df